plugins/modules/aci_l4l7_concrete_device.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Tim Cragg (@timcragg)
+# Copyright: (c) 2025, Shreyas Srish (@shrsr)
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l4l7_concrete_device
+short_description: Manage L4-L7 Concrete Devices (vns:CDev)
+description:
+- Manage Layer 4 to Layer 7 (L4-L7) Concrete Devices.
+options:
+  tenant:
+    description:
+    - The name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  device:
+    description:
+    - The name of the logical device (vns:lDevVip) the concrete device is attached to.
+    type: str
+    aliases: [ device_name, logical_device_name ]
```

Comment:
Is it not better to call it `logical_device` and  add `device` as an alias? It would be less confusing since we have logical device and concrete device but this is just a suggestion

```suggestion
  logical_device:
    description:
    - The name of the logical device (vns:lDevVip) the concrete device is attached to.
    type: str
    aliases: [ device, device_name, logical_device_name ]
```

If you do make the change, it should be applied to every module where logical device is referenced